### PR TITLE
add table creation

### PR DESCRIPTION
**Pull request by KovalenkoOlga**
Add table creation to app

### CODE REVIEW

File: Server/main/routes.js [hunk 1]
```diff
@@ -29,5 +29,16 @@ router.get('/api/get/messages', (req, res, next) => {
         });
 });
 
-
-module.exports = router
+module.exports = {
+    router:router,
+    checkDataBaseState() {
+        pool.query(`SELECT relname FROM pg_class 
+                WHERE relname = 'messages'`,
```
Comment (mkostrikin) [inline Server/main/routes.js]: limit 1

File: Server/main/app.js [hunk 2]
```diff
@@ -7,13 +7,8 @@ var cors = require('cors');
 
 var indexRouter = require('./routes')
 
-
-
-
-
+indexRouter.checkDataBaseState();
```
Comment (n0th1ng-else) [inline Server/main/app.js]: dont forget to handle error in `catch` block :)

File: Server/main/routes.js [hunk 3]
```diff
@@ -2,32 +2,52 @@ var express = require('express')
 var router = express.Router()
 var pool = require('./db')
 
+function runQuery(sql, args ) {
```
Comment (n0th1ng-else) [inline Server/main/routes.js:5]: yesss 👍

File: Server/main/routes.js [hunk 4]
```diff
@@ -2,32 +2,52 @@ var express = require('express')
...
-        values, (q_err, q_res) => {
-            if (q_err) return next(q_err);
-            res.json(q_res.rows);
-        });
+    const sql = `INSERT INTO messages(message, date_created) VALUES($1, NOW())`
+    runQuery(sql, values).then((result)=>{
+        if (result.rows) {
+            res.json(result.rows);
+        }else {return next(result)}
+    })
```
Comment (n0th1ng-else) [inline Server/main/routes.js]: missing `.catch(err => next(err))`

File: Server/main/routes.js [hunk 5]
```diff
@@ -2,32 +2,52 @@ var express = require('express')
...
-    pool.query(`SELECT * FROM messages ORDER BY date_created desc limit $1`,
-        [num], (q_err, q_res) => {
-            if (q_res){
-                res.json(q_res.rows)
-            } else {
-                res.json([])
-            }
-
-        });
+    });
```
Comment (n0th1ng-else) [inline Server/main/routes.js]: missing `.catch(err => next(err))`

**Review by n0th1ng-else — APPROVED** [review]

**Review by mkostrikin — APPROVED** [review]

**Pull request merged**